validate.lic

```diff
@@ -450,6 +450,12 @@ class DRYamlValidator
     warn('***YOU HAVE OUTDATED SETTINGS*** THE SETTING train_in_ap IS NO LONGER USED. READ MORE HERE: https://elanthipedia.play.net/Lich_script_repository#astrology')
   end
 
+  def assert_that_necro_train_first_aid_is_deprecated(settings)
+    return unless settings.thanatology['train_first_aid']
+
+    warn('***YOU HAVE OUTDATED SETTINGS*** THE SETTING thanatology["train_first_aid"] IS NO LONGER USED. READ MORE HERE: https://elanthipedia.play.net/Lich_script_repository#combat-trainer')
```

Comment:
I'd feel better giving the user a more informative message since we know exactly what is wrong (and the `combat-trainer` documentation can be hard to read). What do you think?

Reply:
We could tell them to set the ritual to `dissect` (the FA training ritual) or use cycle